sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.cs

```diff
@@ -125,9 +123,6 @@ public EventProcessor(EventHubOptions options, ITriggeredFunctionExecutor execut
 
             public Task CloseAsync(EventProcessorHostPartition context, ProcessingStoppedReason reason)
             {
-                // signal cancellation for any in progress executions
-                _cts.Cancel();
```

Comment:
I'm a bit confused why we don't still want this cancellation token source. Is it guaranteed that all processing will be done by the time Close is called? It might be safer to leave this token source and used a linked cancellation token in ProcessEventsAsync.